scripts/communityScripts/explore/explore.html

```diff
@@ -286,22 +294,76 @@
 
     <script>
         var exploreComponent = null;
         var locationData = null;
+        var placesData = null;
+
+        function blendedData() {
+            var joinedData = [];
+            if (locationData != null) {
+                for (var i = 0; i < locationData.length; i++) {
+                    joinedData.push({
+                        "index": joinedData.length
+                        , "location": locationData[i].Visit
+                        , "name": locationData[i]['Domain Name']
+                        , "owner": locationData[i].Owner
+                        , "userCount": locationData[i].People
+                        , "description": ""
+                        , "thumbnail": ""
+                    });
+                }
+            }
+
+            if (placesData != null) {
+                var activeThresholdTime = new Date();
+                activeThresholdTime.setMinutes(activeThresholdTime.getMinutes() - 15);
+
+                // while blending metaverse data ensure it is cleaned for active domains
+                for (var i = 0; i < placesData.length; i++) {
+                    if (placesData[i].domain.active && placesData[i].visibility == "open" && new Date(placesData[i].last_activity_update) > activeThresholdTime) {
+                        joinedData.push({
+                            "index": joinedData.length
+                            , "location": placesData[i].address
+                            , "name": placesData[i].name
+                            , "owner": placesData[i].managers[0]
+                            , "userCount": placesData[i].current_attendance
+                            , "description": placesData[i].description
+                            , "thumbnail": placesData[i].thumbnail
+                        });
+                    }
+                }
+            }
+
+            return joinedData;
+        }
+
         function getAddressListData(component) {
             exploreComponent = component;
 
-            if (locationData != null) {
-                exploreComponent.updateData(locationData);
-                return;
+            // if json data not loaded fire emti event for it
+            if (locationData == null) {
+                var readyEvent = {
+                    "action": "requestAddressList",
+                };
+                EventBridge.emitWebEvent(JSON.stringify(readyEvent));
             }
 
-            var readyEvent = {
-                "action": "requestAddressList",
-            };
-            EventBridge.emitWebEvent(JSON.stringify(readyEvent));
+            // if metaverse data not loaded fire a request for it
+            if (placesData == null) {
+                // TODO: fire off data get against the metaverse
+                axios.get("https://metaverse.vircadia.com/live/api/v1/places")
+                    .then((response) => {
+                        placesData = response.data.data.places;
+
```

Comment:
Unnecessary blank line.

Reply:
Updates have been committed to address those comments.